.gitignore

```diff
@@ -5,4 +5,5 @@ package-lock.json
 .rpt2_cache
 .vscode
 yarn-error.log
-.env
\ No newline at end of file
+.env
+yarn.lock
```

Comment:
this can be removed

Reply:
@Adamj1232 updated by reverting the `.gitignore` file